api/build.gradle

```diff
@@ -225,19 +225,18 @@ swaggerSources {
 //
 
 // Generate AppEngine Cron file
+def workbenchEnv = System.getenv('WORKBENCH_ENV') != null ? System.getenv('WORKBENCH_ENV') : 'local'
```

Comment:
nit: you can shorten with groovy elvis operator https://blog.mrhaki.com/2009/08/groovy-goodness-elvis-operator.html